frontend/javascripts/oxalis/controller/scene_controller.js

```diff
@@ -316,12 +315,13 @@ class SceneController {
         }
       }
     } else {
+      const { tdViewDisplayPlanes } = Store.getState().userConfiguration;
```

Comment:
Does changing the user setting trigger a render? Otherwise I would have expected that the setting change will only be applied after the user moved or so.

Reply:
Yes, it still updates when just toggling the setting. I think store changes in general trigger a re-render.